docs/pages/visual-test/Button.js

```diff
@@ -0,0 +1,7 @@
+// @flow strict
+import type { Node } from 'react';
+import { Button } from 'gestalt';
+
+export default function ButtonSpec(): Node {
+  return <Button color="red" text="Save a Pin" />;
```

Comment:
How deep should we go with making sure we're testing every possible visual variation of a component? I would assume we'd want to test everything, but curious about your opinion

Reply:
@dangerismycat we can go as deep as we want but I would start with adding a visual diff test for every component, only after we could add more tests.